src/components/Table/Table.tsx

```diff
@@ -178,9 +178,11 @@ export const Table = chakra(
           sx={styles}
           {...rest}
         >
-          {tableCaption}
-          {columnHeadersElems}
-          {tableBodyElems()}
+          <>
```

Comment:
Why was this necessary?

Reply:
There was a typescript issue. `Table` probably updated in Chakra.